2024-11-03 13:11:49,924 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Current SDK version is 0.16.0
2024-11-03 13:11:49,924 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Configure stats pid to 24468
2024-11-03 13:11:49,925 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Alan de Castro\.config\wandb\settings
2024-11-03 13:11:49,926 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Loading settings from c:\Users\Alan de Castro\Desktop\Projetos\Solar_Anomalies_DL\wandb\settings
2024-11-03 13:11:49,926 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-11-03 13:11:49,926 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-11-03 13:11:49,930 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-11-03 13:11:49,930 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-11-03 13:11:49,931 INFO    MainThread:24468 [wandb_init.py:_log_setup():524] Logging user logs to c:\Users\Alan de Castro\Desktop\Projetos\Solar_Anomalies_DL\wandb\run-20241103_131149-xf2d7v43\logs\debug.log
2024-11-03 13:11:49,932 INFO    MainThread:24468 [wandb_init.py:_log_setup():525] Logging internal logs to c:\Users\Alan de Castro\Desktop\Projetos\Solar_Anomalies_DL\wandb\run-20241103_131149-xf2d7v43\logs\debug-internal.log
2024-11-03 13:11:49,933 INFO    MainThread:24468 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000017FF6221E90>
2024-11-03 13:11:49,941 INFO    MainThread:24468 [wandb_init.py:init():564] calling init triggers
2024-11-03 13:11:49,941 INFO    MainThread:24468 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-11-03 13:11:49,941 INFO    MainThread:24468 [wandb_init.py:init():589] re-initializing run, found existing run on stack: dp5qdte3
2024-11-03 13:11:49,945 INFO    MainThread:24468 [wandb_run.py:_finish():1962] finishing run a-de-castro/Solar_Anomalies/dp5qdte3
2024-11-03 13:11:49,946 INFO    MainThread:24468 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-11-03 13:11:49,947 INFO    MainThread:24468 [wandb_run.py:_restore():2185] restore
2024-11-03 13:11:49,947 INFO    MainThread:24468 [wandb_run.py:_restore():2191] restore done
2024-11-03 13:11:55,627 INFO    MainThread:24468 [wandb_run.py:_footer_history_summary_info():3760] rendering history
2024-11-03 13:11:55,628 INFO    MainThread:24468 [wandb_run.py:_footer_history_summary_info():3792] rendering summary
2024-11-03 13:11:55,642 INFO    MainThread:24468 [wandb_run.py:_footer_sync_info():3719] logging synced files
2024-11-03 13:11:56,014 INFO    MainThread:24468 [wandb_init.py:init():614] starting backend
2024-11-03 13:11:56,014 INFO    MainThread:24468 [wandb_init.py:init():618] setting up manager
2024-11-03 13:11:56,025 INFO    MainThread:24468 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-11-03 13:11:56,025 INFO    MainThread:24468 [wandb_init.py:init():624] backend started and connected
2024-11-03 13:11:56,059 INFO    MainThread:24468 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-11-03 13:11:56,059 INFO    MainThread:24468 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-11-03 13:11:56,059 INFO    MainThread:24468 [wandb_init.py:init():716] updated telemetry
2024-11-03 13:12:08,091 INFO    MainThread:24468 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-11-03 13:12:08,690 INFO    MainThread:24468 [wandb_run.py:_on_init():2254] communicating current version
2024-11-03 13:12:08,865 INFO    MainThread:24468 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.18.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-11-03 13:12:08,866 INFO    MainThread:24468 [wandb_init.py:init():800] starting run threads in backend
2024-11-03 13:12:09,109 INFO    MainThread:24468 [wandb_run.py:_console_start():2233] atexit reg
2024-11-03 13:12:09,109 INFO    MainThread:24468 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-11-03 13:12:09,109 INFO    MainThread:24468 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-11-03 13:12:09,110 INFO    MainThread:24468 [wandb_run.py:_redirect():2178] Redirects installed.
2024-11-03 13:12:09,111 INFO    MainThread:24468 [wandb_init.py:init():841] run started, returning control to user process
2024-11-03 13:12:09,116 INFO    MainThread:24468 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-11-03 13:12:09,116 INFO    MainThread:24468 [wandb_init.py:_pause_backend():435] pausing backend
2024-11-03 13:12:09,181 INFO    MainThread:24468 [wandb_init.py:_resume_backend():440] resuming backend
2024-11-03 13:12:13,399 INFO    MainThread:24468 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-11-03 13:12:13,400 INFO    MainThread:24468 [wandb_init.py:_pause_backend():435] pausing backend
2024-11-03 13:13:18,253 INFO    MainThread:24468 [wandb_init.py:_resume_backend():440] resuming backend
2024-11-03 13:13:36,895 INFO    MainThread:24468 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-11-03 13:13:36,896 INFO    MainThread:24468 [wandb_init.py:_pause_backend():435] pausing backend
2024-11-03 13:13:36,995 INFO    MainThread:24468 [wandb_init.py:_resume_backend():440] resuming backend
2024-11-03 13:13:36,999 INFO    MainThread:24468 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-11-03 13:13:36,999 INFO    MainThread:24468 [wandb_init.py:_pause_backend():435] pausing backend
2024-11-03 13:13:37,039 INFO    MainThread:24468 [wandb_init.py:_resume_backend():440] resuming backend
2024-11-03 13:13:37,079 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Current SDK version is 0.16.0
2024-11-03 13:13:37,080 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Configure stats pid to 24468
2024-11-03 13:13:37,081 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Alan de Castro\.config\wandb\settings
2024-11-03 13:13:37,081 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Loading settings from c:\Users\Alan de Castro\Desktop\Projetos\Solar_Anomalies_DL\wandb\settings
2024-11-03 13:13:37,081 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-11-03 13:13:37,081 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-11-03 13:13:37,100 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-11-03 13:13:37,101 INFO    MainThread:24468 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-11-03 13:13:37,101 INFO    MainThread:24468 [wandb_init.py:_log_setup():524] Logging user logs to c:\Users\Alan de Castro\Desktop\Projetos\Solar_Anomalies_DL\wandb\run-20241103_131337-mpdv22qr\logs\debug.log
2024-11-03 13:13:37,101 INFO    MainThread:24468 [wandb_init.py:_log_setup():525] Logging internal logs to c:\Users\Alan de Castro\Desktop\Projetos\Solar_Anomalies_DL\wandb\run-20241103_131337-mpdv22qr\logs\debug-internal.log
2024-11-03 13:13:37,101 INFO    MainThread:24468 [wandb_init.py:init():564] calling init triggers
2024-11-03 13:13:37,102 INFO    MainThread:24468 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-11-03 13:13:37,102 INFO    MainThread:24468 [wandb_init.py:init():589] re-initializing run, found existing run on stack: xf2d7v43
2024-11-03 13:13:37,106 INFO    MainThread:24468 [wandb_run.py:_finish():1962] finishing run a-de-castro/Solar_Anomalies/xf2d7v43
2024-11-03 13:13:37,107 INFO    MainThread:24468 [jupyter.py:save_history():445] not saving jupyter history
2024-11-03 13:13:37,107 INFO    MainThread:24468 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-11-03 13:13:37,107 INFO    MainThread:24468 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-11-03 13:13:37,108 INFO    MainThread:24468 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-11-03 13:13:37,108 INFO    MainThread:24468 [wandb_run.py:_restore():2185] restore
2024-11-03 13:13:37,109 INFO    MainThread:24468 [wandb_run.py:_restore():2191] restore done
2024-11-03 13:13:41,918 INFO    MainThread:24468 [wandb_run.py:_footer_history_summary_info():3760] rendering history
2024-11-03 13:13:41,918 INFO    MainThread:24468 [wandb_run.py:_footer_history_summary_info():3792] rendering summary
2024-11-03 13:13:41,927 INFO    MainThread:24468 [wandb_run.py:_footer_sync_info():3719] logging synced files
